autosynth/github.py

```diff
@@ -353,6 +369,47 @@ def update_pull_labels(
             labels=list(label_names),
         )
 
+    def list_repos(self, org: str) -> List[str]:
+        """Returns a list of all the repositories in an organization.
+
+        Args:
+            org (str): The name of the organization.
+
+        Returns:
+            List[str]: The list of repository names.
+        """
+        url = f"{_GITHUB_ROOT}/orgs/{org}/repos?type=public"
```

Comment:
You probably also want to filter out archived repositories

Reply:
Done.